Tests/TrackerRadarKitTests/Resources/mockTrackerData.json

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
Don't necessarily need to keep this file any more. I wrote it to provide easily digestible mock data since debugging against the huge `trackerData.json` file was impossible, but happy to remove it.